gdal/plan.sh

```diff
@@ -1,21 +1,21 @@
 pkg_name=gdal
 pkg_origin=core
-pkg_version=2.2.1
+pkg_version=2.4.0
 pkg_description="GDAL is a translator library for raster and vector geospatial data formats"
 pkg_upstream_url=http://www.gdal.org/
 pkg_license=('MIT')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source=http://download.osgeo.org/gdal/${pkg_version}/gdal-${pkg_version}.tar.gz
-pkg_shasum=61837706abfa3e493f3550236efc2c14bd6b24650232f9107db50a944abf8b2f
+pkg_shasum=a568cf3dc7bb203ae12a48e1eb2a42302cded499ef6eccaf9e8f09187d8ce75a
 pkg_build_deps=(
   core/gcc
   core/make
   core/pkg-config
   core/patchelf
 )
 pkg_deps=(
-  core/glibc
   core/gcc-libs
```

Comment:
Whats the reason for the dep order change here?

Reply:
I made it alphabetical

Reply:
In general, we prefer not to reorder deps for alphabetization purposes (see [this thread](https://github.com/habitat-sh/core-plans/pull/1786#discussion_r211034574) for details), though more for PATH than for libraries or headers.  In this case though, we know glibc and gcc-libs work well together regardless of order.